cmake_modules/IcebergThirdpartyToolchain.cmake

```diff
@@ -285,3 +287,20 @@ function(resolve_nlohmann_json_dependency)
 endfunction()
 
 resolve_nlohmann_json_dependency()
+
+# ----------------------------------------------------------------------
+# zlib
+
+function(resolve_zlib_dependency)
+  # use system zlib, zlib is required by arrow and avro
+  find_package(ZLIB REQUIRED)
+  if(ZLIB_FOUND)
+    list(APPEND ICEBERG_SYSTEM_DEPENDENCIES ZLIB)
+    message(STATUS "ZLIB_FOUND")
```

Comment:
Can we use a single `message` call?

Reply:
BTW, we should also check if target `ZLIB::ZLIB` exists

Reply:
i found those code in zlib cmake script
    add_library(ZLIB::ZLIB STATIC IMPORTED)
    set(ZLIB_LIBRARIES ${ZLIB_STATIC_LIB})
    set(ZLIB_INCLUDE_DIRS "${ZLIB_PREFIX}/include")
    set_target_properties(ZLIB::ZLIB PROPERTIES IMPORTED_LOCATION ${ZLIB_LIBRARIES})
so i think there is no need to check it.